idaes/power_generation/flowsheets/subcritical_power_plant/subcritical_power_plant.py

```diff
@@ -0,0 +1,1850 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+
+import time as wall_clock
+import matplotlib.pyplot as plt
+
+import pyomo.environ as pyo
+from pyomo.network import Arc
+from idaes.core import FlowsheetBlock
+from idaes.generic_models.properties import iapws95
+from idaes.core.util.model_statistics import degrees_of_freedom
+import idaes.core.util.scaling as iscale
+from idaes.power_generation.properties import FlueGasParameterBlock
+from idaes.power_generation.control.pid_controller import PIDController
+import idaes.power_generation.flowsheets.subcritical_power_plant.subcritical_boiler_flowsheet as blr
+import idaes.power_generation.flowsheets.subcritical_power_plant.steam_cycle_flowsheet as stc
+from idaes.core.util.dyn_utils import (
+    copy_values_at_time,
+    copy_non_time_indexed_values)
+import idaes.logger as idaeslog
+import os
+import idaes.core.util.tables as tables
+from idaes.core.util.misc import svg_tag
+
+_log = idaeslog.getLogger(__name__)
+
+__author__ = "Boiler Subsystem Team (J. Ma, M. Zamarripa)"
+
+
+def _new_solve(self, model, **kwargs):
+    self.options["nlp_scaling_method"] = "user-scaling"
+    self.options["linear_solver"] = "ma27"
+    self.options["tol"] = 1e-6
+    self.options['ma27_pivtol'] = 0.01
+    self.options['ma27_pivtolmax'] = 0.6
+    if kwargs["tee"]:
```

Comment:
This is causing the failing test.  I thought we were going to remove this.  If not change this line to:

```python
    if kwargs.get("tee", False):
```

Reply:
Let's drop this weirdness once PR #180 gets merged.